packages/authentication/README.md

```diff
@@ -1,3 +1,49 @@
 # @loopback/authentication
 
 A LoopBack component for authentication support.
+
+# Overview
+ It demonstrates how to use LoopBack's user models and passport to interact with other authentication providers.
+ User can login using a passport.js strategy, which could include a third party provider.
+
+
+# Installation
+
+Then in your Node.js project root, run:
+
+```shell
+npm install -S @loopback/authentication
```

Comment:
IIRC, `-S` is for `--save-dev`, that's not right. This component will be typically used in production, also not many people are familiar with `-s` vs. `-S`. Let's use full `--save` please.